ui/shared/utils/constants.js

```diff
@@ -861,6 +861,31 @@ export const VEP_CONSEQUENCE_ORDER_LOOKUP = ORDERED_VEP_CONSEQUENCES.reduce((acc
   ({ ...acc, [consequence.value]: i }),
 {})
 
+export const SVTYPE_TEXT_MAPPING = ORDERED_VEP_CONSEQUENCES.reduce((acc, consequence) =>
```

Comment:
this works, but I think it is more maintainable to make a list of just the sv types rather than filtering the whole consequence list here. So above, define 
```
SV_TYPES = [
  {
    description: 'A large duplication',
    text: 'Duplication',
    value: 'DUP',
  }, 
 ...
]
```
Then inside `ORDERED_VEP_CONSEQUENCES` add `...SV_TYPES.map(o => ({ ...o, group: VEP_GROUP_SV})` to add all the SVs to the main list of consequences, and then for the mapping do 
```
SV_TYPES.reduce((acc, { value, text }) = ({ ...acc, [value]: text }), {})
```
I might also rename `SVTYPE_TEXT_MAPPING` to `SVTYPE_LOOKUP` to be more consistent with the other constant names